libqtile/backend/x11/window.py

```diff
@@ -506,6 +500,7 @@ def __init__(self, window, qtile):
         self.float_y: int | None = None
         self._float_width: int = self._width
         self._float_height: int = self._height
+        self.previous_layer = (False, False, True, False, False, False)
```

Comment:
Would it be helpful to our future selves to add a comment here listing what these are?